README.md

```diff
@@ -16,3 +16,47 @@ The project and Python dependencies can be installed from source with:
 ```
 pip3 install .
 ```
+
+## Configuration
+
+Many EduLinq tools share a common configuration system.
+This system provides a consistent way to supply configuration options to the CLI tool being used.
+While each CLI tool may require additional options, the configuration loading process is the same across the EduLinq ecosystem.
+
+By default, the config file is named `edq-config.json`.
+This is customizable and may differ depending on the tool being used.
+
+For the purposes of this documentation we are going to use the default config file name.
+
+### Configuration Sources
+
+Configuration options can come from several places, with later sources overwriting earlier ones.
+
+#### Global Configuration
+
+The default place a global config is looked for is `<platform-specific user config location>/edq-config.json`.
+This is considered to be the "proper" place to store user-related configuration, according to [platformdirs](https://github.com/tox-dev/platformdirs).
+The location where a global config will be looked for can be changed by passing a path to `--global-config` through the command line.
+This type of config is best suited for login credentials or persistent user preferences.
+Run any CLI tool with `--help` to see the exact path on your platform.
+
+#### Local Configuration
+
+Local configuration files can be found in different locations.
+The first file found will be used, and other locations will not be searched.
+
+##### Search order for a local config file:
```

Comment:
I don't see any need to create these subsection.
They can all be under the parent.